### PR TITLE
fix: ignore white space

### PR DESCRIPTION
**Pull request by ws-cd**

### CODE REVIEW

File: src/Editor.tsx [hunk 1]
```diff
@@ -148,25 +146,37 @@ function InlineEditor() {
...
-                  node.type === "button",
-                mode: "all",
-              });
+              const windSelection = window.getSelection();
+              const text = windSelection?.toString();
 
-              Transforms.wrapNodes(editor, button, { split: true });
-              Transforms.collapse(editor, { edge: "end" });
+              if (text && selection && editor && text.trimEnd().length) {
+                editor.selection = {
```
Comment (heruiwoniou) [inline src/Editor.tsx:161]: The selection property contains the user's current selection, if any. Don't set it directly; use Transform.select

File: src/Editor.tsx [hunk 2]
```diff
@@ -148,25 +146,37 @@ function InlineEditor() {
...
               };
-              Transforms.unwrapNodes(editor, {
-                at: [],
-                match: (node, path) =>
-                  !Editor.isEditor(node) &&
-                  "type" in node &&
-                  node.type === "button",
-                mode: "all",
-              });
+              const windSelection = window.getSelection();
```
Comment (heruiwoniou) [inline src/Editor.tsx:157]: Why use this to get selection text?

**Review by heruiwoniou — APPROVED** [review]

File: src/Editor.tsx (hunk 1 above)
Comment (ws-cd) [reply src/Editor.tsx:161]: I will try to use it

Comment (heruiwoniou) [reply src/Editor.tsx:161]: ☺️

File: src/Editor.tsx (hunk 2 above)
Comment (heruiwoniou) [reply src/Editor.tsx:157]: You can try this `Eidtor.getFragment`

**Pull request merged**